exercises/concept/logs-logs-logs/.meta/design.md

```diff
@@ -20,3 +20,21 @@ This exercise's prerequisites Concepts are:
 - `strings`
 - `switch-statement`
 - `constructors`
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer][analyzer]:
+
+1. Parse log level
+    - `LogLine.getLogLevel()`
+       - `actionable`: Verify that a switch statement was used. If they used if-else, comment that switch is better due to there being many cases.
+2. Support unknown log level
+    - `essential`: Verify that a default was used in a switch statement. Comment as previously stated in the case of an if-else.
+3. Convert log line to short format
+    - `essential`: Verify that there exists a field dedicated to the encoded log level.
+    - `actionable`: Said field should be private and final (this will then require the use of a getter for the log level!).
```

Comment:
I think it would be sufficient to check that they have used a field and is `final` (no need to check if `private` and that there is a method to get the log level).